app/src/main/java/com/example/banking/data/repository/AccountsRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.banking.data.repository
```

Comment:
repository interface should place on domain layer (otherwise this abstraction doesn't make sense)

Reply:
fixed